docs/custom-flows/overview.mdx

```diff
@@ -1,5 +1,65 @@
 ---
-sanity_slug: /docs/authentication/custom-flows
+title: Custom flows
+description: Learn the process behind building custom sign-up and sign-in flows with Clerk.
 ---
 
-# This is a stub
+# Custom flows
+
+If Clerk's [prebuilt components](/docs/components/overview) don't meet your specific needs or if you require more control over the authentication flow, Clerk enables you to build fully custom sign-up and sign-in flows using our [`useSignUp()`](/docs/references/react/use-sign-up) and [`useSignIn()`](/docs/references/react/use-sign-in) React hooks.
+
+But before diving deep into the code side, it is important to first understand the process behind building sign-up and sign-in flows, which fields are required, and the necessary steps to authenticate a user.
+
+## Sign-up flow
+
+Clerk provides a flexible way to build sign-up flows in your application. You can use a single [`SignUp`](/docs/references/javascript/sign-up/sign-up) object to gather information, verify their email address or phone number, add OAuth accounts, and finally, convert them into a [`User`](/docs/references/javascript/user/user).
```

Comment:
Suggestion: Since the links repeat a lot you could use https://www.markdownguide.org/basic-syntax/#reference-style-links

This way you'd e.g. write

```md
[`SignUp`][sign-up-ref]

blabla...

[sign-up-ref]: /docs/references/javascript/sign-up/sign-up
```